key-vault-node-quickstart-v3/README.md

```diff
@@ -0,0 +1,192 @@
+    ```
+    When the web app has been created, the Azure CLI shows output similar to the following example:
+    ```
+    {
+      "availabilityState": "Normal",
+      "clientAffinityEnabled": true,
+      "clientCertEnabled": false,
+      "cloningInfo": null,
+      "containerSize": 0,
+      "dailyMemoryTimeQuota": 0,
+      "defaultHostName": "<app_name>.azurewebsites.net",
+      "enabled": true,
+      "deploymentLocalGitUrl": "https://<username>@<app_name>.scm.azurewebsites.net/<app_name>.git"
+      < JSON data removed for brevity. >
+    }
+    ```
+    Browse to your newly created web app and you should see a functioning web app. Replace <app_name> with a unique app name.
+
+    ```
+    http://<app name>.azurewebsites.net
+    ```
+    The above command also creates a Git-enabled app which allows you to deploy to azure from your local git. 
+    Local git is configured with url of 'https://<username>@<app_name>.scm.azurewebsites.net/<app_name>.git'
+
+- Create a deployment user
+    After the previous command is completed you can add add an Azure remote to your local Git repository. Replace <url> with the URL of the Git remote that you got from Enable Git for your app.
+
+    ```
+    git remote add azure <url>
+    ```
+
+## Enable Managed Service Identity
+
+Azure Key Vault provides a way to securely store credentials and other keys and secrets, but your code needs to authenticate to Key Vault to retrieve them. Managed Service Identity (MSI) makes solving this problem simpler by giving Azure services an automatically managed identity in Azure Active Directory (Azure AD). You can use this identity to authenticate to any service that supports Azure AD authentication, including Key Vault, without having any credentials in your code.
+
+Run the assign-identity command to create the identity for this application:
+
+```azurecli
+az webapp identity assign --name <app_name> --resource-group "<YourResourceGroupName>"
+```
+
+This command is the equivalent of going to the portal and switching **Managed service identity** to **On** in the web application properties.
+
+### Assign permissions to your application to read secrets from Key Vault
+
+Write down or copy the output of the command above. It should be in the format:
+        
+        {
+          "principalId": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
+          "tenantId": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
+          "type": "SystemAssigned"
+        }
+        
+Then, run this command using the name of your Key Vault and the value of PrincipalId copied from above:
+
+```azurecli
+az keyvault set-policy --name '<YourKeyVaultName>' --object-id <PrincipalId> --secret-permissions get
+```
+
+## Deploy the Node App to Azure and retrieve the secret value
+
+Now that everything is set. Run the following command to deploy the app to Azure
+
+```
+git push azure master
+```
+
+After this when you browse https://<app_name>.azurewebsites.net you can see the secret value.
```

Comment:
Instead of "After this..."

When the git push command has completed you can now navigate to https://<app_name>.azurewebsites.net to see the secret value.